app/static/src/app/components/header/AppHeader.vue

```diff
@@ -18,7 +18,10 @@
         <li><router-link id="all-sessions-link" class="dropdown-item" to="/sessions">All Sessions</router-link></li>
       </ul>
     </span>
-    <version-menu v-if="initialised" :wodin-version="wodinVersion"></version-menu>
+    <span v-if="initialised" style="display: flex; align-items: center;">
+      <version-menu :wodin-version="wodinVersion"></version-menu>
+      <language-switcher :languagesKeys="languagesKeys"/>
```

Comment:
I think this should be the other order (so switcher to the left of version)

Reply:
done!